byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -51,23 +85,29 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> places = new ArrayList<>();
+    ArrayList<String> places = new ArrayList<String>();
     for (PlaceType location : PlaceType.values()) {
-      String place = formatLocation(location);
-      places.add(place);
+      if (!nonInterests.contains(location.toString())) {
+        String place = formatPlace(location.toString());
+        places.add(place);
+      }
     }
+    for (String customInterest : customInterests) {
```

Comment:
You can use `list1.addAll(list2)` instead of having to iterate through the entire arraylist.

Reply:
At first, I didn't do this since I wanted to keep `customInterests` in the same format as the other lists (i.e. underscores as spaces, non-capitalized, etc.), and then format them later in similar fashion. I guess since they are being hard-coded anyways, I can format it and use this method to avoid formatting through a function call.

Fixed